### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by ErikEJ**
Related to #2237

### CODE REVIEW

**Comment by ErikEJ** [comment]
@AndriySvyryd FYI

**Comment by ErikEJ** [comment]
Oh, are you planning to publish this also?  EntityFramework.SqlServerCompact

File: src/EntityFramework.NuGet/EntityFramework.NuGet.nuspec [hunk 1]
```diff
@@ -54,5 +55,6 @@
     <file src="buildTransitive\**\*" target="buildTransitive/" />
     <file src="content\**\*" target="content/" />
     <file src="tools\**\*" target="tools/" />
+    <file src="../../README.md" target="\" />
```
Comment (AndriySvyryd) [inline src/EntityFramework.NuGet/EntityFramework.NuGet.nuspec:58]: Please make the slashes consistent in this file

**Comment by ErikEJ** [comment]
So add a readme to SqlCe package also?

**Comment by AndriySvyryd** [comment]
> So add a readme to SqlCe package also?

That would be great. Even a single sentence will do.

**Comment by ErikEJ** [comment]
@AndriySvyryd Add SqlCe readme and fixed all the slashes

**Comment by ErikEJ** [comment]
@AndriySvyryd I also sharpened the new provider readme slightly, as per the discussion [here](https://github.com/dotnet/ef6/issues/2237#issuecomment-2120915779)

**Review by AndriySvyryd — APPROVED** [review]

**Comment by AndriySvyryd** [comment]
Thanks for your contribution!

**Pull request merged**